ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -579,18 +610,58 @@ Check Environment Variables Exist
     # Add and Run JupyterLab Code Cell in Active Notebook    import os
     FOR    ${idx}   ${env_variable_dict}    IN ENUMERATE    @{exp_env_variables}
         Remove From Dictionary    ${env_variable_dict}     k8s_type    input_type
-        # IF    "${input_type}" == "${KEYVALUE_TYPE}"
         ${n_pairs}=    Get Length    ${env_variable_dict.keys()}
         FOR  ${pair_idx}   ${key}  ${value}  IN ENUMERATE  &{env_variable_dict}
             Log   ${pair_idx}-${key}-${value}
             Run Keyword And Continue On Failure     Run Cell And Check Output    import os;print(os.environ["${key}"])    ${value}
```

Comment:
(optionally) Another way to obtain the env vars could be using the keyword `Run Command In Container` executing echo $MYVAR. It could be a new keyword in the same file `Get Environment Variable In Container`

Reply:
makes sense, thank you. However I would like to focus on other tasks for the remaining time of the sprint so I will not implement this suggestion in this PR.